content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -1,117 +1,179 @@
+
+However, with a larger box, you also need to include more packing material to keep the vases from breaking during shipping, which takes up precious space in the box. Likewise, with a larger node size, you can get a lot of transaction data, but the data may also contain `rippled` start up information and ledger reports, which takes up precious RAM that can't be used for the subset of data you're actually using.
+
+A part of tuning these configurations is about choosing the right node size so that the subset of data you're actually using can be retrieved in the fewest trips to disk storage, while also ensuring that all of it, including other data you aren't using right now, can fit in your available RAM.
+
+Ripple recommends you always use the largest node size your available RAM can support. See the following table for recommended settings.
 
 #### Recommendation
 
-| Available RAM for `rippled` | `node_size` value | Notes                              |
-|:----------------------------|:------------------|:-----------------------------------|
-| < 8GB                       | `tiny`            | Not recommended                    |
-| 8GB                         | `small`           |                                    |
-| 16GB                        | `medium`          |                                    |
-| 32GB                        | `huge`            | Recommended for production servers |
+Each `node_size` has a corresponding requirement for available RAM. For example, if you set `node_size` to `huge`, you should have at least 32GB of available RAM to help ensure that `rippled` can run smoothly.
+
+To tune your server, it may be useful to start with `tiny` and increase the size to `small`, `medium`, and so on as you refine the requirements for your use case.
+
+***TODO: Small thing, but I switched the order of the node size and RAM columns because the node size column contains the value that you are setting and should be the starting point of focus.***
+
+| `node_size` value | Available RAM for `rippled` | Notes                      |
+|:------------------|:----------------------------|:---------------------------|
+| `tiny`            | < 8GB                       | Not recommended for testing or production servers. If not specified in `rippled.cfg`, `tiny` is the default value. |
+| `small`           | 8GB                         | Recommended for test servers. |
+| `medium`          | 16GB                        | Note that the `rippled-example.cfg` has its `node_size` value set to `medium`. ***TODO: The doc also says: With a `node_size` of `medium`, a `rippled` server can be reasonably stable in a test Linux system with 16GB of RAM -- is our recommendation for test servers - small or medium? Or either?*** |
```

Comment:
We don't have a strict recommendation for test servers. A test server is whichever server you have around to test things with.

You can say that the default config uses this value. (The RPM installation literally copies and renames the "example config" to become the default config.)